CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.1](https://github.com/brave/brave-ios/releases/tag/v1.57.1)
+
+- Resolved issue where Brave Ads are disabled if Brave News is enabled already. ([#8098](https://github.com/brave/brave-ios/issues/8098)) 
```

Comment:
The `already` placed in the wrong spot. I think the following would probably sound better:

```
- Resolved issue where Brave Ads are disabled if Brave News was already enabled. ([#8098](https://github.com/brave/brave-ios/issues/8098)) 
```